source/core/parameters_multiphysics.cc

```diff
@@ -553,6 +559,20 @@ Parameters::CahnHilliard::parse_parameters(ParameterHandler    &prm,
         throw(std::runtime_error("Invalid epsilon setting strategy. "
                                  "Options are 'automatic' or 'manual'."));
 
+      const std::string op_epsilon_verbosity = prm.get("verbosity");
+      if (op_epsilon_verbosity == "quiet")
+        {
+          CahnHilliard::epsilon_verbosity = Parameters::EpsilonVerbosity::quiet;
+        }
+      else if (op_epsilon_verbosity == "verbose")
+        {
+          CahnHilliard::epsilon_verbosity =
+            Parameters::EpsilonVerbosity::verbose;
+        }
+      else
+        throw(std::runtime_error("Invalid epsilon verbosity. "
+                                 "Options are 'quiet' or 'verbose'."));
```

Comment:
Use the AssertThrow macro instead please

Reply:
By looking at the code I realized we actually don't need the last part (else{ throw error}) because deal.ii already deals with the string not matching a given pattern. So we actually never enter the else. 
I think this is the same with all the other parameters that take strings as input, maybe we could clean up all of them in a subsequent PR.
If you think the "else" should be kept for a reason, please tell me

Reply:
IRL discussion lead to leaving the exception throwing for an extra-security. Question solved